### ISSUE TITLE
[Keyboard Navigation - Azure Stream Analytics - Overview]: Expand and collapse button of "Query" text box under "Overview" blade are not accessible through keyboard.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1420]
**Blade name**: Microsoft_Azure_StreamAnalytics > QueryBlade

**User Experience:**
If the control is not accessible through keyboard then the end user who rely on keyboard might not be able to perform any action related to that functionality using keyboard and might face difficulties while interacting with "Query" text box.

**Repro Steps:**
1. Open https://ms.portal.azure.com/ and login with HCL subscription.
2. Search for "Azure stream analytics" and select it.
3. Navigate to the any created  "Azure stream analytics" and select it.
4. Navigate to the various controls of the "Overview" blade
5. Navigate on to "Query" text box and try to activate "Expand/collapse" button.
6. Observe whether "Expand/collapse" button is accessible through keyboard or not.

**Actual Result:​**
Expand/collapse button of "Query" text box under "Overview" blade is not accessible through keyboard.
​
**Expected Result**:​
Expand/collapse button of "Query" text box under "Overview" blade should be accessible through keyboard.

**MAS Reference:**
MAS 2.1.1 - Keyboard

**Test Environment:** 
Microsoft windows version 1809(OS Build 17763.253)
Microsoft Edge 44.17763.1.0

![MAS2 1 1EditBoxMonaco](https://user-images.githubusercontent.com/30456910/56345097-927ee580-61dc-11e9-971f-bca1bbbd9e18.png)

**Comment by isidorn** [comment]
@sonimishra hi if you think this is an issue in the monaco editor please file the issue in concise steps in humand readable form only mentioning the monaco editor. Do not mention the Azure Stream Analyitics in your reproducable steps. Please put some effor in this.
Thank you for your understanding.
